python_purify/__init__.py

```diff
@@ -1,3 +1,3 @@
-from core import ImagePurify
-from core import WordPurify
-from core import VideoPurify
+from .core import ImagePurify
```

Comment:
Without these changes you get the following error:

Traceback (most recent call last):
```
...
...
...
    from python_purify import WordPurify
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/python_purify-1.2.3-py3.6.egg/python_purify/__init__.py", line 1, in <module>
ModuleNotFoundError: No module named 'core'
```

Reply:
https://stackoverflow.com/questions/34753206/init-py-cant-find-local-modules/34753235

Reply:
please add `from __future__ import absolute_import` to the top of this file. should work for py2 and py3